assets/src/components/media/manager/MediaManager.tsx

```diff
@@ -121,6 +122,7 @@ export interface MediaManagerState {
   showDetails: boolean;
   error: Maybe<string>;
   filteredMimeTypes: string[] | undefined;
+  uploading: boolean;
```

Comment:
Added a new "Uploading File" message during uploads. Before, the user could still interact with the dialog during uploads, which could cause inconsistent results.